src/terminal/message.rs

```diff
@@ -0,0 +1,34 @@
+use super::emoji;
+use console::Emoji;
+
+fn message(label: Emoji, msg: &str) {
+    println!("{}", format!("{0} {1} {0}", label, msg));
+}
+
+pub fn info(msg: &str) {
+    message(emoji::INFO, msg);
+}
+
+pub fn success(msg: &str) {
+    message(emoji::SPARKLES, msg);
+}
+
+pub fn warn(msg: &str) {
+    message(emoji::WARN, msg);
+}
+
+pub fn user_error(msg: &str) {
+    message(emoji::EYES, msg);
+}
+
+pub fn service_error(msg: &str) {
```

Comment:
Unused -- makes clippy complain.

Reply:
There isn't currently a place where these would be used with a println!, only with failure::bail. I agree keeping them would be ideal, i kinda want the equivalent of an eslintignore here though.

Reply:
there is #[allow(dead_code)]

Reply:
i do not think we should commit dead code. i think the pattern here is clear enough that someone can add. if there's a space in the code to use it, put it in that one spot.

it's very hard to keep track of and clean up the allow dead code attributes